### DISCUSSION TITLE
make subfigure caption subreference default to itself

### DISCUSSION THREAD

**Discussion by QiangF** [kjambunathan/org-mode-ox-odt#159] [Ideas]
Currently to add a sub index, we have to specify "#+NAME" and in the short caption "#+CAPTION[[[name]]]:" to refer to that name, and the name appears twice, which seems unecessary: 

    #+CAPTION: Animals
    #+ATTR_ODT: :category "figure" :list-table t
    -
      -
        #+NAME: a
        #+CAPTION[[[a]]]: A Mythical Beast
        #+ATTR_ODT: :width 7.1
        [[./org-mode-unicorn.png]]
      -
        #+NAME: b
        #+CAPTION[[[b]]]: A Mythical Beast
        [[./org-mode-unicorn.png]]

Why not omit the NAME in "#+CAPTION[[[name]]]:" if there is a "#+NAME:"

    #+CAPTION: Animals
    #+ATTR_ODT: :category "figure" :list-table t
    -
      -
        #+NAME: a
        #+CAPTION: A Mythical Beast
        #+ATTR_ODT: :width 7.1
        [[./org-mode-unicorn.png]]
      -
        #+NAME: b
        #+CAPTION: A Mythical Beast
        [[./org-mode-unicorn.png]]

**Comment by kjambunathan** [comment]
> Currently to add a sub index, we have to specify "#+NAME" and in the short caption "#+CAPTION[[[name]]]:" to refer to that name, and the name appears twice, which seems unecessary:



**_In your case, what you say is true._**    It is not how short caption is meant to be used.   This is precisely what I added   a `short-caption-as-label` as an EXPERMENTAL feature in `org-odt-experimental-features`.  

I prefer to NOT change the official definition of what a short caption is.  **_Anything can go in a a short caption, and in most cases it  WON'T be a label reference_**


This screenshot illustrates the purpose of short caption 
![Screenshot from 2022-02-13 07-16-02](https://user-images.githubusercontent.com/19869864/153735027-8fe39b20-a618-477d-9464-90902cf8f8dd.png)
